content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -47,8 +47,8 @@ Verify that zero padding is enabled on the PowerFlex storage pools that will be
 ### Install PowerFlex Storage Data Client
 
 The CSI Driver for PowerFlex requires you to have installed the PowerFlex Storage Data Client (SDC) on all Kubernetes nodes which run the node portion of the CSI driver. 
-SDC could be installed automatically by CSI driver install on Kubernetes nodes with OS platform which support automatic SDC deployment; for Red Hat CoreOS (RHCOS), RHEL 7.9 and RHEL 8.x. On Kubernetes nodes with OS version not supported by automatic install, you must perform the Manual SDC Deployment steps [below](#manual-sdc-deployment).
-Refer to https://hub.docker.com/r/dellemc/sdc for supported OS versions.
+SDC could be installed automatically by CSI driver install on Kubernetes nodes with OS platform which support automatic SDC deployment; for Red Hat CoreOS (RHCOS), RHEL 7.9, RHEL 8.4, RHEL 8.6. On Kubernetes nodes with OS version not supported by automatic install, you must perform the Manual SDC Deployment steps [below](#manual-sdc-deployment).
```

Comment:
I would not include specific versions here. Can we link to SDC docs instead? We'd have to update each release otherwise

Reply:
Some customers and internal teams are facing the issue using the unsupported versions. So I am adding the specific supported versions. Thanks